include/application/tasks/valve_demo.h

```diff
@@ -0,0 +1,19 @@
+/**
+ * This is a toy task for actuating a valve and demonstrating
+ * the platform's valve management functionality. This task
+ * does the following:
+ * 
+ *  1. Opens the inspiratory valve.

```

Comment:
could we have a step 0 where we wait without sending any signal to the valves. 

The goal of doing this would be to verify that when not actuated/initially the valves are open.

Reply:
I can change the state machine to include this. Good idea.

Reply:
![image](https://user-images.githubusercontent.com/1484030/97645724-efcb3800-1a0a-11eb-81b4-3ceb736551cf.png)

Updated. There is now period at the beginning of each state machine cycle where all valves are closed.